rskj-core/src/main/java/co/rsk/net/light/LightSyncProcessor.java

```diff
@@ -305,27 +293,46 @@ public void startFetchRound() {
         //Starting fetch sub chains
     }
 
-    public void differentFirstBlocks() {
-
+    public void differentFirstBlocks(LightPeer lightPeer) {
```

Comment:
we should sort publics methods on the top and then the private ones

Reply:
Methods sorted better